howmanypeoplearearound/__main__.py

```diff
@@ -147,27 +146,23 @@ def scan(adapter, scantime, verbose, number, nearby, jsonprint, out, allmacaddre
     for line in output.decode('utf-8').split('\n'):
         if verbose:
             print(line)
-        if len(line.strip()) == 0:
+        if not line.strip():
             continue
         mac = line.split()[0].strip().split(',')[0]
         dats = line.split()
-        if len(dats) == 3:
-            if ':' not in dats[0] or len(dats) != 3:
-                continue
+        if len(dats) == 3 and ':' in dats[0]:
             if mac not in foundMacs:
                 foundMacs[mac] = []
-            rssi = 0
-            rssi = float(dats[2].split(',')[0]) / 2 + \
-                float(dats[2].split(',')[0]) / 2
-            foundMacs[mac].append(rssi)
+            rssi = float(dats[2].split(',')[0]) / 2
```

Comment:
This was my mistake, that needs to be corrected - could you introduce it?

It should be:
```python
rssi = float(dats[2].split(',')[0]) / 2 + float(dats[2].split(',')[1]) / 2
```
 basically averaging the two values.